index.js

```diff
@@ -378,6 +383,14 @@ export const PullRequest = {
     const variables = { id: nodeId };
     await graphql(query, variables);
   },
+  async createComment({ self, args }) {
+    const { name: owner } = self.match(root.users.one);
+    const { name: repo } = self.match(root.users.one.repos.one);
+    const { number } = self.match(root.users.one.repos.one.issues.one);
+    const { body, commitId, path, position } = args;
```

Comment:
In general we should not do any "pointing" other than using refs. When using this action, a user can point to a specific commit and file with arguments. Instead, the ability to comment on a commit should be in a node of type `Commit`. Same with `File`